index.src.html

```diff
@@ -1401,24 +1401,21 @@
 
 - Client processing, and the {{ClientData}} structure, for registration extensions and signature extensions.
 
-- Authenticator processing, and the <a>authenticatorData</a> structure, for signature extensions.
+- Authenticator processing, and the <a>authenticatorData</a> structure, for registration extensions and signature extensions.
```

Comment:
we need to add "registration extension" and "signature extension" (whcih should perhaps be "assertion extn" or "authn extn") to the terminology section.